supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,53 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Technical best practices
+
+* Use hyperlinks consistently.
+* Do not use non hyperlinks.
```

Comment:
I see two negatives here :exploding_head: . What does this mean? If possible, an example can be useful.